tahoe_sites/api.py

```diff
@@ -116,3 +119,41 @@ def is_active_admin_on_organization(user, organization):
         is_active=True,
         is_admin=True,
     ).exists()
+
+
+def create_tahoe_site(organization, site):
+    """
+    Link a site to an organization. Together they are a tahoe-site
+
+    :param organization: Organization object to be linked
+    :param site: Site object to be linked
+    :return: TahoeSite object
+    """
+    if zd_helpers.should_site_use_org_models():
+        organization.sites.add(site)
+        return None
+    return TahoeSite.objects.create(organization=organization, site=site)
```

Comment:
This probably should be an internal `helpers.py` function.

Reply:
no more create by link existing